medicai/models/upernet/upernet.py

```
@@ -0,0 +1,273 @@
+            allowed_families=UPerNet.ALLOWED_BACKBONE_FAMILIES,
+        )
+        inputs = encoder.input
+        spatial_dims = len(input_shape) - 1
+        pyramid_outputs = encoder.pyramid_outputs
+
+        # For UPerNet, encoder depth should be 5.
+        if encoder_depth not in (3, 4, 5):
+            raise ValueError(
+                f"To build {self.__class__.__name__}, encoder_depth must be 3, 4 or 5. "
+                "We need at least 1 bottleneck and 3 lateral features for FPN."
+            )
```

Ruff (0.14.2)

Avoid specifying long messages outside the exception class

(TRY003)